core/src/main/codegen/templates/Parser.jj

```diff
@@ -7335,9 +7336,13 @@ SqlNode StandardFloorCeilOptions(Span s, boolean floorFlag) :
         }
     )?
     <RPAREN> {
-        SqlOperator op = floorFlag
-            ? SqlStdOperatorTable.FLOOR
-            : SqlStdOperatorTable.CEIL;
+        SqlOperator op;
+        boolean isBigQuery = this.conformance.semantics() == SqlLibrary.BIG_QUERY;
```

Comment:
look at way to make more scaleable

Reply:
can you create a function in SqlStdOperatorTable similar to `SqlStdOperatorTable.like(boolean,boolean)`? Its args should be boolean and conformance.

Reply:
What does the boolean argument represent in this case? Whether you want it to be like/unlike the conformance you pass in?

Or do you mean you pass two conformances in (the one being used and the one you're checking against) and you return `BOOLEAN` (whether they are the same or not)?